.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, windows-latest, macos-latest]
+    steps:
+      - run: git config --global submodule.fetchJobs 8
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.15.0
+      - uses: actions/upload-artifact@v3
+        with:
+          path: ./wheelhouse/*.whl
```

Comment:
So, we upload the artifact to the wheelhouse, but what do with it? Should we add a release action? Should we try TestPyPI first?

Reply:
https://github.com/pypa/cibuildwheel/blob/e8bb208374117ef8e54e28f8fada054fbe701cc8/examples/github-deploy.yml#L44-L63